frontend/src/components/MapView/LeftPanel/LeftPanelTabs/index.tsx

```diff
@@ -0,0 +1,308 @@
+  createStyles({
+    root: {
+      display: 'flex',
+      flexDirection: 'row',
+      height: '100%',
+      overflowX: 'hidden',
+      overflowY: ({ tabValue }) =>
+        tabValue === 1 || tabValue === 3 ? 'hidden' : 'auto',
+    },
+    tabsWrapper: {
+      display: 'flex',
+      flexDirection: 'column',
+      height: '100%',
+    },
+    tabsContainer: {
+      backgroundColor: '#566064',
+      order: -2,
+      width: ({ panelSize }) =>
+        panelSize !== PanelSize.folded ? PanelSize.medium : PanelSize.folded,
+    },
+    indicator: {
+      backgroundColor: '#53888F',
+      height: '10%',
+    },
+    tabRoot: {
+      textTransform: 'none',
+      minWidth: 50,
+      maxWidth: '50%',
+    },
+    tabSelected: {
+      opacity: 1,
+      backgroundColor: '#3C3F40',
+    },
+  }),
+);
+
+interface TabsProps {
+  layersPanel: React.ReactNode;
+  chartsPanel: React.ReactNode;
+  analysisPanel: React.ReactNode;
+  tablesPanel: React.ReactNode;
+  areTablesAvailable: boolean;
+  panelSize: PanelSize;
+  resultsPage: JSX.Element | null;
+  setPanelSize: React.Dispatch<React.SetStateAction<PanelSize>>;
+}
+
+const LeftPanelTabs = memo(
+  ({
+    layersPanel,
+    chartsPanel,
+    analysisPanel,
+    tablesPanel,
+    areTablesAvailable,
+    panelSize,
+    resultsPage,
+    setPanelSize,
+  }: TabsProps) => {
+    const { t } = useSafeTranslation();
+    const dispatch = useDispatch();
+    const tabValue = useSelector(leftPanelTabValueSelector);
+    const activeLayers = useSelector(activeLayersSelector);
+    const classes = useStyles({ panelSize, tabValue });
+
+    const areChartLayersAvailable = useMemo(() => {
+      return getWMSLayersWithChart().length > 0;
+    }, []);
+
+    const handleChange = useCallback(
+      (_: any, newValue: number) => {
+        setPanelSize(PanelSize.medium);
+        dispatch(setTabValue(newValue));
+      },
+      [dispatch, setPanelSize],
+    );
+
+    const renderedLayersTabLabel = useMemo(() => {
+      if (
+        tabValue !== 0 &&
+        panelSize !== PanelSize.folded &&
+        activeLayers.length >= 1
+      ) {
+        return (
+          <Badge
+            anchorOrigin={{
+              horizontal: 'left',
+              vertical: 'top',
+            }}
+            badgeContent={activeLayers.length}
+            color="secondary"
+          >
+            <LayersOutlined style={{ verticalAlign: 'middle' }} />
+            <Box ml={1}>{t('Layers')}</Box>
+          </Badge>
+        );
+      }
+      return (
+        <>
+          <LayersOutlined style={{ verticalAlign: 'middle' }} />
+          <Box ml={1}>{t('Layers')}</Box>
+        </>
+      );
+    }, [activeLayers.length, panelSize, t, tabValue]);
+
+    const a11yProps = useCallback((index: any) => {
+      return {
+        id: `full-width-tab-${index}`,
+        'aria-controls': `full-width-tabpanel-${index}`,
+      };
+    }, []);
+
+    const renderedTabWidth = useMemo(() => {
```

Comment:
You could simply do something like:

`const nTabs = 2 + sum(reChartLayersAvailable, areTablesAvailablem)`
return `'calc(100% / nTabs)`

Reply:
Yes you are right on this changing it.